testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -2074,11 +2074,20 @@ def test_frame_interval_convention():
         universe1 = MDAnalysis.Universe(PSF, DCD)
         array1 = universe1.trajectory.timeseries(skip=10)
         universe2 = MDAnalysis.Universe(PSF, DCD, in_memory=True,
-                                        in_memory_frame_interval=10)
+                                        in_memory_step=10)
         array2 = universe2.trajectory.timeseries()
         assert_equal(array1, array2,
                      err_msg="Unexpected differences between arrays.")
 
+    @staticmethod
+    def test_slicing():
+        universe = MDAnalysis.Universe(PDB_small, DCD)
+        # Skip only the last frame
+        universe.transfer_to_memory(start=0, stop=-1)
```

Comment:
Could you suggest what values would work in our case? I'm asking because I have no idea what the actual value will be, so I don't know what to compare against.